### PR TITLE
Implements update_document_type_form.

### PR DESCRIPTION
**Pull request by ashercodeorg**
Tested manually.

### CODE REVIEW

File: bin/solr/update_document [hunk 1]
```diff
@@ -27,6 +28,44 @@ def get_solr_id_from_db_id(db_id)
...
 end
 
+def update_document_type_form(db_id, solr_id)
+  # TODO(asher): Refactor this and the code in bin/cron/process_forms to a
+  # shared method.
+  db_form = DB[:forms].where(id: db_id).first
+
+  kind = Object.const_get(db_form[:kind])
+  begin
+    solr_doc = JSON.load(db_form[:data]).
```
Comment (tanyaparker) [inline bin/solr/update_document]: Should we name this solr_form_doc to be more consistent with the update user method?

File: bin/solr/update_document [hunk 2]
```diff
@@ -27,6 +28,44 @@ def get_solr_id_from_db_id(db_id)
...
+        user_i: db_form[:user_id],
+        parent_db_form_i: db_form[:parent_id]
+      })
+    solr_doc = kind.index(solr_doc) if kind.respond_to?(:index)
+
+    address_to_search = solr_doc['school_address_s'].to_s.strip
+    address_to_search = solr_doc['zip_code_s'].to_s.strip if address_to_search.empty?
+    address_to_search = db_form[:created_ip].to_s.strip if address_to_search.empty?
+    location = Geocoder.search(address_to_search).first
+    if location
```
Comment (tanyaparker) [inline bin/solr/update_document]: Might not be worth it if you're going to refactor w/ bin/cron/process_forms, but could we factor out the location stuff from both these methods?

**Comment by tanyaparker** [comment]
Two unblocking thoughts. Otherwise lgtm

**Comment by tanyaparker** [comment]
👍

File: bin/solr/update_document (hunk 1 above)
Comment (ashercodeorg) [inline bin/solr/update_document]: Done (in the PR against `staging-next`).

File: bin/solr/update_document (hunk 2 above)
Comment (ashercodeorg) [inline bin/solr/update_document]: Lumping that work in the TODO (that I do intend to resolve).

**Pull request closed without merging**